deployments/linkerd.go

```diff
@@ -178,3 +187,49 @@ func (k Linkerd) Upgrade(ctx context.Context, c *kubernetes.Cluster, ui *termui.
 
 	return k.apply(ctx, c, ui, options, true)
 }
+
+func (k Linkerd) createLinkerdJob(ctx context.Context,
+	c *kubernetes.Cluster,
+	jobName,
+	serviceAccountName,
+	imageName,
+	jobCommand string) error {
+
+	backoffLimit := int32(1)
+
+	job := batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: jobName,
+		},
+		Spec: batchv1.JobSpec{
+			Template: corev1.PodTemplateSpec{
+				Spec: corev1.PodSpec{
+					ServiceAccountName: serviceAccountName,
+					Containers: []corev1.Container{
+						{
+							Name:            jobName,
+							Image:           imageName,
+							ImagePullPolicy: "Always",
```

Comment:
This will download the image every time. I don't think that's wanted?

Reply:
Changed it

Reply:
Removed it.